### ISSUE TITLE
Add alt text to logos and icons

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/rockarch.org#47]
- Homepage logo and logo on other pages that link to the homepage should have alt text (`alt= Rockefeller Archive Center`). 
- Icons on homepage require alt text. Because they are decorative, the alt text should be null so that a screenreader can ignore them (`alt=""`). 
- Images on the About the Rockefellers page require alt text. The family portrait images are decorative, so the image can instead have null alt text (`alt=""`).
